openkfw/TruBudget
# 878 forgot password feature

### Checklist




### Description
* added forgot password link to login screen
* added sending email with reset password link
* added password reset screen

### How to test

1. run app locally in --full mode with email service and either set up smtp user that would sent email to your address, or just run mailslurper to catch emails
2. login with any user, set email address and log out
3. On login screen click Forgot password link
4. put in the email address that you set for the user
5. if it is correct address you should receive an email with reset password link either to your inbox or in mailslurper
6. Open the received link and set new password, auth token is currently set to expire after 15 minutes
7. try logging in with new password
8. try disabling your user and try going to forgot password again
9. email for disabled user shouldn't be sent

Closes #878
